backend/src/controllers/v1/exercisesController.js

```diff
@@ -1,5 +1,11 @@
 import { initModels as model } from '../../models'
+import { RecordNotFound } from './../../lib/errors'
 
+
+export const setUser = async id => {
```

Comment:
Please why are you export this function?

Reply:
ok initially I wanted to use it in the user controller as well. I will move it to the exercise controller